tinygrad/codegen/uops.py

```diff
@@ -289,7 +289,7 @@ def print(self):
     for i,u in enumerate(self):
       print(f"{i:4d} {str(u.op):20s}: {str(u.dtype) if u.dtype is not None else '':25s} " f"{str([self.uops.index(x) for x in u.src]):32s} {u.arg}")
 
-  def graph_rewrite(self, sink, pm):
+  def graph_rewrite(self, sink: UOp, pm: PatternMatcher):
```

Comment:
we don't have a strict style guide, but i'd remove the spaces in between like other functions in this file

Reply:
Is there a way to have the linter check this?

Reply:
ok

Reply:
is this  a rule for only type annotations or also eg. lambda function